charts/opentelemetry-kube-stack/examples/cloud-demo/values.yaml

```diff
@@ -0,0 +1,42 @@
+clusterName: demo
+opentelemetry-operator:
+  enabled: true
+defaultCRConfig:
+  presets:
+    otlpExporter:
+      enabled: true
+      endpoint: ingest.example.com:443
+      headers:
+        "access-token": "${ACCESS_TOKEN}"
```

Comment:
at first i liked the idea of providing an OTLP exporter preset, but now i'm not sure... interested in other peoples thoughts!

Reply:
I think no.  Configuring OTLP exporters for your favorite backend are going to vary to widely to make this useful I think.